unit_tests/testExpTables/testExpTables.f90

```diff
@@ -195,22 +194,20 @@ PROGRAM testExpTables
   ENDDO
   DO i=-10000,0
     bool = (reftbl(i) .APPROXEQ. testET2(5)%table(i))
+    bool=.FALSE.
```

Comment:
Uh, come again?  Are we wanting to fail all 10001 iterations of this loop?

Reply:
woops

Reply:
yeah, so I added that to see what was actually printing...completely worthless FINFO...no idea why it did what it was doing...and it was a defect :-/